### PR TITLE
Update no_resources.md

### PR DESCRIPTION
**Pull request by korbinib**
unmerged changes from #1223, lost in #1257

### CODE REVIEW

File: pages/national_resources/no_resources.md [hunk 1]
```diff
@@ -242,6 +242,21 @@ Sikt is a Norwegian governmental agency that provides [(sensitive) personal data
 
 NRIS, a collaboration between Sigma2 (a subsidiary of Sikt) and four universities, gives [technical and administrative support](https://www.sigma2.no/user-support) to researchers in Norway on large-scale data storage and high-performance computing.
 
+
+### Communities for research data management support personell
+
+* [Life-Science RDM Group](https://elixir.no/organization/biomeddata/Life-Science-RDM-Group)
+* [RDA Norway](https://www.rd-alliance.org/groups/rda-norway)
+* [University of Oslo Data Manager Network](https://www.ub.uio.no/english/libraries/dsc/data-managers-network/)
+* [Bergen Research Data Network](mailto:research-data@uib.no?subject=forskningsdata@uib.no)
```
Comment (Nazeeefa) [inline pages/national_resources/no_resources.md:251]: For Bergen Research Data Network, do you mean to link this https://www.uib.no/en/research/143544/research-infrastructure-network ? here, two email addresses are mentioned..

Comment (korbinib) [reply pages/national_resources/no_resources.md:251]: no, this is 2 different networks

File: pages/national_resources/no_resources.md [hunk 2]
```diff
@@ -242,6 +242,21 @@ Sikt is a Norwegian governmental agency that provides [(sensitive) personal data
 
 NRIS, a collaboration between Sigma2 (a subsidiary of Sikt) and four universities, gives [technical and administrative support](https://www.sigma2.no/user-support) to researchers in Norway on large-scale data storage and high-performance computing.
 
+
+### Communities for research data management support personell
+
```
Comment (Nazeeefa) [inline pages/national_resources/no_resources.md:247]: Maybe an introductory sentence could be useful here f.ex something along the lines of "Norway has several RDM communities that are created to bring together data management support staff/personnel through knowledge-exchange activities and discussions on specific DM topics" (now that I have read 'personell', I am not sure which spelling is correct. I was only familiar with 'personnel')

File: pages/national_resources/no_resources.md [hunk 3]
```diff
@@ -167,7 +167,7 @@ ref_to_main_resources:
 ## Introduction
 
 This page provides an overview of the data management resources in Norway. The target audience is the Norwegian scientific community in the life sciences and collaborators.
-The [Data Stewardship Wizard instance from ELIXIR Norway](https://elixir-no.ds-wizard.org/knowledge-models/elixir.no:lifesciences-elixir-norway:latest/preview) provides an interactive way to navigate these recommendations and resources.
+The [Data Stewardship Wizard instance from ELIXIR Norway](https://elixir-no.ds-wizard.org/knowledge-models/elixir.no:lifesciences-elixir-norway:latest/preview) provides an interactive way to navigate these recommendations and resources. You can also find condensed information in the interlinked [RDM LookUp from ELIXIR Norway](elixir.no/rdm-lookup/)
```
Comment (Nazeeefa) [inline pages/national_resources/no_resources.md:170]: Does the link 'elixir.no/rdm-lookup/' need https:// to be recognised as link in markdown file?

**Review by Nazeeefa — CHANGES_REQUESTED** [review]
Thanks. Added few suggestions, hope you find them helpful

File: pages/national_resources/no_resources.md (hunk 2 above)
Comment (korbinib) [reply pages/national_resources/no_resources.md:247]: Nowegian spelling vs English fixed now

File: pages/national_resources/no_resources.md [hunk 4]
```diff
@@ -242,6 +242,21 @@ Sikt is a Norwegian governmental agency that provides [(sensitive) personal data
 
 NRIS, a collaboration between Sigma2 (a subsidiary of Sikt) and four universities, gives [technical and administrative support](https://www.sigma2.no/user-support) to researchers in Norway on large-scale data storage and high-performance computing.
 
+
+### Communities for research data management support personell
```
Comment (korbinib) [inline pages/national_resources/no_resources.md:246]:
```suggestion
### Communities for research data management support personnel

There are several RDM communities (with subject or local focus) in Norway to enable exchange and discussion:
 
```

**Review by korbinib — COMMENTED** [review]
please check again

**Review by Nazeeefa — APPROVED** [review]

**Comment by Nazeeefa** [comment]
Thank you so much @korbinib

**Pull request merged**